css-borders-4/Overview.bs

```diff
@@ -361,7 +361,77 @@ as well as account for sharp edges and overlaps.
 Issue: 'border-radius' already handles *adjacent* corners overlapping by shrinking the radiuses proportionally.
 A negative ''superellipse()'' parameter allows for *opposite* corners to sometimes overlap, and needs additional restrictions defined.
 
-Issue <a href="https://github.com/w3c/csswg-drafts/issues/11610">#11610</a>: check if we need additional rendering restrictions.
+<h5 id=corner-shape-border-rendering>
+Effect of 'corner-shape' on border rendering</h5>
+
+When rendering a [=border=] for a box that has a 'border-radius' and a 'corner-shape'
+the inner border follows the curve of the outer shape, with a nearly consistent distance from the outer path throughout,
+or a linearly increasing distance if the 'border-width' at the different edges is different.
+
+<div algorithm="adjust-border-inner-path-for-corner-shape">
+To compute the <dfn>adjusted untrimmed inner top-left corner starting point</dfn>:
+
+	1. Let |s| be the [=superellipse parameter=] of the element's 'corner-top-left-shape'.
+	1. Let |w| be the [=used value=] of the element's 'border-left-width'.
+	1. If |s| is equal or greater than 1, return <code>|w|, 0</code>.
+	1. If |s| is equal or less than -1, return <code>0, -|w|</code>.
+
+		Note: Because of the above clamping, when the shape is more convex than a ''corner-shape/round'' or more concave than a ''corner-shape-scoop'',
+		the inner curve cannot would appear to have a "belly" towards the middle, as it cannot remain constant while maintaining the same curvature.
+	1. Let |k| be <code>0.5<sup>|s|</sup></code>.
+	1. Let |normalizedHalfCorner| be <code>0.5<sup>|k|</sup></code>.
+	1. Let |offset| be <code>2 - &radic;2</code>.
+	1. Let |adjustedX| be <code>max(0, 2 * |normalizedHalfCorner| - |offset|)</code>.
+	1. Let |adjustedY| be <code>max(0, 2 * (1 - |normalizedHalfCorner|) - |offset|)</code>.
+	1. Return <code>|adjustedX|, |adjustedY|</code>.
+</div>
+
+<div algorithm="corner-shape-path">
+To compute the inner path of a corner:
+
+	1. Let |adjustedX|, |adjustedY| be the result of the [=adjusted untrimmed inner top-left corner starting point=] steps,
+		modified to consider the corresponding 'border-width' and 'corner-shape' values at the start and end of a path`.
+	1. Create a path based on the [=canonical superellipse formula=], with the outer corner as the center, starting from |adjustedX| and ending at |adjustedY|.
+		User agents MAY create an approximation of this path rather than try to follow the formula precisely, for performance reasons.
+	1. Trim the resulting path so that it starts at a point that has a 'border-width' distance from the outer.
+</div>
+
+<div class=note>
+The superellipse quadrant is scaled so that:
+	* Its center stays at the outer corner.
+	* Its start and end point are perpendicular to the outer starting point.
+	* Its distance from the outer curve is the corresponding 'border-width'.
+
+This is illustrated in the following chart:
+<figure>
+	<img src="images/corner-shape-border.svg"
```

Comment:
This could use some details of where the points/etc calculated by the above algorithms are. 

Is the hull calculated in the above algorithms in some way?